ibverbs/src/lib.rs

```diff
@@ -938,6 +941,8 @@ impl<'res> PreparedQueuePair<'res> {
     /// Internally, this uses `ibv_modify_qp` to mark the `QueuePair` as initialized
     /// (`IBV_QPS_INIT`), ready to receive (`IBV_QPS_RTR`), and ready to send (`IBV_QPS_RTS`).
     /// Further discussion of the protocol can be found on [RDMAmojo].
+    /// 
+    /// If the endpoint contains a Gid, the routing will be global.
```

Comment:
Let's also write out what the implication is of this in terms of what fields will be set on `ah_attr`.